en_US/configuration/limiter.md

```diff
@@ -0,0 +1,45 @@
+# Limiter
```

Comment:
I think those are better known as `Rate Limiters` in other systems, and by using that name it would become more clear what it does more quickly.